res/values-fr/strings_mapis.xml

```diff
@@ -14,7 +14,6 @@
         name="show_supported_apps_list_summary">Choix des applications depuis lesquelles il faut scrobbler</string>
     <string
         name="supported_apps_category_title">Applications activées</string>
-    <string name="scrobble_droid_supported_apps">Applications supportées par Scrobble Droid</string><!-- Unused for now. See : ScrobbleDroidMusicReceiver.java -->
```

Comment:
Maybe you could just <!-- --> block it out.
It really should be made translatable....